index.html

```diff
@@ -4,7 +4,293 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo&family=Josefin+Sans:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
     <title>Travel With Us</title>
   </head>
-  <body></body>
+  <body>
+    <main>
+      <section class="section1">
+        <h1 class="section1__title section1_title">Bhromaon</h1>
+        <p class="section1__subtitle section1_subtitle">Safe Travel Anywhere</p>
+        <div class="input_position">
+          <input
+            class="section1__input section1_input"
+            type="text"
+            placeholder="Search Your Travel Destination"
+          />
+          <button class="section1__button section1_button" type="submit">
+            <svg
+              width="16"
+              height="16"
+              viewBox="0 0 16 16"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M15.7824 13.833L12.6666 10.7177C12.5259 10.5771 12.3353 10.499 12.1353 10.499H11.6259C12.4884 9.39596 13.001 8.00859 13.001 6.49937C13.001 2.90909 10.0914 0 6.50048 0C2.90959 0 0 2.90909 0 6.49937C0 10.0896 2.90959 12.9987 6.50048 12.9987C8.00996 12.9987 9.39756 12.4863 10.5008 11.6239V12.1332C10.5008 12.3332 10.5789 12.5238 10.7195 12.6644L13.8354 15.7797C14.1292 16.0734 14.6042 16.0734 14.8948 15.7797L15.7793 14.8954C16.0731 14.6017 16.0731 14.1267 15.7824 13.833ZM6.50048 10.499C4.29094 10.499 2.50018 8.71165 2.50018 6.49937C2.50018 4.29021 4.28781 2.49976 6.50048 2.49976C8.71001 2.49976 10.5008 4.28708 10.5008 6.49937C10.5008 8.70852 8.71314 10.499 6.50048 10.499Z"
+                fill="white"
+              />
+            </svg>
+            Search
+          </button>
+        </div>
+        <p class="section1__scroll section1_scroll">Scroll Down</p>
+        <svg
+          class="section1__scroll-svg section1_scroll-svg"
+          width="14"
+          height="16"
+          viewBox="0 0 14 16"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M6.46966 11.9211L0.396437 5.84786C0.103531 5.55496 0.103531 5.08008 0.396437 4.78721L1.10478 4.07886C1.39719 3.78646 1.87109 3.7859 2.16419 4.07761L7 8.89077L11.8358 4.07761C12.1289 3.7859 12.6028 3.78646 12.8952 4.07886L13.6035 4.78721C13.8964 5.08011 13.8964 5.55499 13.6035 5.84786L7.53034 11.9211C7.23744 12.214 6.76256 12.214 6.46966 11.9211Z"
+            fill="white"
+          />
+        </svg>
+      </section>
+      <header class="header">
+        <h2 class="header__title">Bhromaon</h2>
+        <ul class="header__list">
+          <li>Who We Are</li>
+          <li>What We Do</li>
+          <li>Travel With Us</li>
+          <li>Contact Us</li>
+        </ul>
+      </header>
+      <section class="section2">
```

Comment:
краще називати не `.section2`,` .section3` і т.д., а орієнтуватись на вміст напр.:
```suggestion
      <section class="why-choose-us">
```